### ISSUE TITLE
soanrC# plugin 

### ISSUE THREAD

**Issue by rexuekonglong** [SonarSource/sonar-csharp#1034]
### Description
Sensor C# Properties [csharp]   
Property missing: 'sonar.cs.analyzer.projectOutPaths'. No protobuf files will be loaded for this project.
0/0 source files have been analyzed

### Repro steps

1.my  C# project property  (sonarqube 6.3)   then  I  upgrade sonarqube to v6.7.1
   sonar.projectKey = Preferential_net
   sonar.projectName = Preferential_net
   sonar.projectVersion = 1.0
   sonar.sources = .
   sonar.language = cs

2. what's   'sonar.cs.analyzer.projectOutPaths'  mean?  
    Does it mean  the C# project code need to be compiled  ?

3. What the hole sonarC#  projcet  property  is   ?

### Related information

* SonarC# Version  6.7 (build 4627)
* Sonarqube Version 6.7
* Sonar-scanner Version 3.0.3

**Comment by daguiler** [comment]
Hi, we have just updated this plugin to 6.7 build 4627 and getting same issue. Some projects are analyzed but others are not (this error shows), but not sure what the condition is so far.

**Comment by daguiler** [comment]
I can confirm this.
We're getting this error since [6.7.0.4267](https://github.com/SonarSource/sonar-csharp/releases/tag/6.7.0.4267).
I can also confirm this doesn't happen with [6.6.0.3969](https://github.com/SonarSource/sonar-csharp/releases/tag/6.6.0.3969).
There is a pre-release in between, but did not test it.

**Comment by valhristov** [comment]
@rexuekonglong are you using Scanner for MSBuild or just Scanner CLI? The latter is not directly supported and could generate the errors you mentioned. The only way to analyze .NET projects is through the Scanner for MSBuild.

@daguiler could you please run the Scanner for MSBuild with /D:sonar.verbose=true and send us the logs, it would be of great help diagnosing the problem.

**Comment by rexuekonglong** [comment]
@valhristov  I user the Sacnner CLI  3.0.3 all the time .  When sonarqube is 6.3.1 Scanner CLI work, then I  upgrade  sonarqube to 6.7.1 and the sonarC# plugin ,  Scanner CLI  can't  just analyze .NET  code .

**Comment by valhristov** [comment]
To analyze .NET code you have to use Scanner for MSBuild. The analysis is faster and more precise than older versions with Scanner CLI, however you will not be able to analyze non-.NET languages in the same time. There is no way to use Scanner CLI with SonarC# 6+.

**Comment by daguiler** [comment]
Attaching the logs @valhristov.

I noticed this warning is occurring for all of our test projects, for example from line 3384 to line 3477 (BusinessLogic.UnitTests).

Also, and this is probably a different issue, I don't see any analysis information for some of the projects, like from line 3478 to line 4389 (Bss). I'm seeing many compilation warnings for that project in the log (lines 2192 to 2253), but I don't see them listed in SonarQube afterwards. Specifically, following lines caught my eye:
```
09:50:33.191 INFO: Sensor C# Properties [csharp]
09:50:33.191 DEBUG: Analyzer working directory 'D:\src\corporate-portal\build\.sonarqube\out\4\output-cs' contains 6 .pb file(s)
09:50:33.191 DEBUG: Found Roslyn issues report
09:50:33.191 INFO: Sensor C# Properties [csharp] (done) | time=0ms
```
Even though it says it detected 6 .pb files, I don't see any code issues in SonarQube for this project.

A similar project, for which I do see code issues, is the project from line 4491 to line 4890 (DataAccess). This one seems to work ok.

[analize-back-end-log.txt](https://github.com/SonarSource/sonar-csharp/files/1597651/analize-back-end-log.txt)

**Comment by valhristov** [comment]
Thanks for the detailed log @daguiler.

The test projects' results were silently not uploaded to SonarQube before, you could safely ignore those warnings for now. We will add full support for test project analysis and unit test results in the near future.

Regarding the main projects, I see that you are using `subst` to create a new drive. I suspect that the current folder is changed before (or during) the build and the paths to the files in the `*.dll.RoslynCA.json` files do not match the paths that were gathered in the begin step. It is required to run the scanner and the build from the same folder. Could you try to `subst` and change the current folder before running the begin step?

**Comment by daguiler** [comment]
mm I'll give it try, but not sure because using the same script with SonarC# 6.6 correctly yields results from that project

**Comment by valhristov** [comment]
I am closing this ticket, please let me know if you still have trouble running the analysis.

**Comment by czerwinski1977** [comment]
@valhristov you were saying "There is no way to use Scanner CLI with SonarC# 6+." -- Until the update to SonarQube 6.7.1, SonarC# 6.7.1 I was also happily running analyses using SonarQube 6.5.0, Scanner 3.0.3 and SonarC# 6.6.0.
Thus, I find it difficult to understand your statement. Could you explain please?
In addition, can anybody shed light on the sonar.cs.analyzer.projectOutPaths parameter?
Thank you!

**Comment by valhristov** [comment]
That's the ticket that drops the MSBuild 12 support - #374, it was released with SonarC# 6.0. Without the code in it, it should be impossible to analyze without Scanner for MSBuild (that's a breaking change, hence the major version bump).

**Comment by TheGrussalo** [comment]
@valhristov I have exactly the same issue:

15:19:21.456 WARN: Property missing: 'sonar.cs.analyzer.projectOutPaths'. No protobuf files will be loaded for this project.
15:19:21.449 DEBUG: '[Deprecated] C# Integration Tests Coverage Report Import' skipped because one of the required properties is missing
15:19:21.461 WARN: No roslyn issues report not found for this project.

Running Sonar 6.7.1 (migrated from 5.6.3 to 5.6.7 then to 6.7.1) with plugins:
    sonar-csharp-plugin-6.8.1.4648
    sonar-javascript-plugin-4.1.0.6085
    sonar-ldap-plugin-2.2.0.608

I am using the sonar scanner for msbuild version 4.0.2.892.

Note:  This is the first time I have used the sonar scanner for msbuild as previously we used the sonar runner.

I have the output in debug mode but corporate policy is stopping me from uploading it to here, is there any other way I can send you the file?

**Comment by valhristov** [comment]
@TheGrussalo what version of MSBuild are you using? Later versions of Scanner and SonarC# do not support MSBuild 12 and earlier.

**Comment by TheGrussalo** [comment]
@valhristov I'm running 15.5.180.51428

The commands I'm using are:
        SonarQube.Scanner.MSBuild.exe begin /d:sonar.verbose=true /k:"*******.*****.NugetPackageManagement" /n:"******* ******* ***** Nuget Package Management" /v:"1.0" 
        "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\msbuild.exe" *******.*****.NugetPackageManagement.sln  /t:Rebuild 
        SonarQube.Scanner.MSBuild.exe end

**Comment by valhristov** [comment]
The `projectOutPaths` property is not written when the analyzed project language is not C# or VB.NET. Are there more errors in the logs?

Could you share the logs with me? I have opened [a file request](https://www.dropbox.com/help/files-folders/create-file-request) on my Dropbox where you could upload the logs:
https://www.dropbox.com/request/CszJAO1ZNXFVl1gwo5hA 

Also it would be nice if you could run msbuild.exe with the /v:diag switch and upload that log too.

The multi-language projects are not supported for the time being, you will have to analyze your files twice (once for C# with Scanner for MSBuild and once for JS with the normal Scanner CLI) to two separate SonarQube projects. We will be working on fixing this inconvenience in the future, but I cannot say when at this moment.

**Comment by TheGrussalo** [comment]
@valhristov I uploaded some files yesterday, let me know if anything else would help.

**Comment by toanvo729** [comment]
@valhristov : Hi valhristov, we got the same problem with C# project. It did not generate an output-cs folder: "Analyzer working directory does not exist. Analyzer results won't be loaded from this directory". And we're using Sonarqube 7.0 and run SonarQube.Scanner.MSBuild to analyze. Please advise

**Comment by Evangelink** [comment]
@TheGrussalo @toanvo729 Could you open (each one of you) a separate issue for your problem. Usually there are small differences and it gets hard to follow when mixing logs and answers.

Thanks

**Comment by milindkarpe** [comment]
SonarQube dashboard is not showing any errors or bugs.

- Sonar-scanner cli version - 3.0.3.778
- SonarQube version - 6.7.0.33306
- Visual Studio - 2013

Have added 1 python and 1 C# code file in my source folder & ran the Sonar scanner.

SonarQube displays the bugs of python file however does not display any errors of c# file.

**Comment by valhristov** [comment]
@milindkarpe, as I mentioned before in this thread, scanning .NET source code with Scanner CLI is not supported; you need to use the Scanner for MSBuild.

The general recommendation for multi-language projects is to scan them separately and have two SonarQube projects - one for .NET and one for the others. However, if you add the Python (or JavaScript, etc.) to your .NET project, they should be automatically picked up by the Scanner for MSBuild. The other language files must be referenced by the .NET project, if they are just present in the same folder it will not work.

**Comment by milindkarpe** [comment]
@valhristov  Thanks for reply!

- Sonar-ScannerMsbuild - 4.0.2.892
- Visual Studio - 2013

Have installed the MSBuild 14 on machine and ran the Sonar-Scanner for MSbuild and its working.

**Comment by bbi-yggy** [comment]
I had a similar problem that proved to be an MSBuild compatibility issue. Using MSBuild from the Microsoft.NET Framework folder didn't work; using the one from Visual Studio 2017 did.

Didn't work:
C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe

Worked:
C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\MSBuild.exe

**Comment by rexuekonglong** [comment]
"2017\Professional\MSBuild\15.0\Bin\MSBuild.exe" is the only build tools for VS2017 since .net framework 4.7

At 2018-07-25 05:21:49, "Yossarian King" <notifications@github.com> wrote:


I had a similar problem that proved to be an MSBuild compatibility issue. Using MSBuild from the Microsoft.NET Framework folder didn't work; using the one from Visual Studio 2017 did.

Didn't work:
C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe

Worked:
C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\MSBuild.exe

**Comment by yfrindel** [comment]
I've recently upgraded our SonarQube server from a 5.x version to a 7.2 version. I've launched the database migration like it is described in the documentation, and the server seems correctly upgraded.
The projects was correctly analyzed with the previous version of SonarQube, but now with the 7.2 version I have these following error messages :

    2018-08-16T07:43:28.8310428Z ##[error]08:43:28.822 WARN: Property missing: 'sonar.cs.analyzer.projectOutPaths'. No protobuf files will be loaded for this project.
    2018-08-16T07:43:28.8311036Z 08:43:28.822 WARN: Property missing: 'sonar.cs.analyzer.projectOutPaths'. No protobuf files will be loaded for this project.
    2018-08-16T07:43:28.8313994Z ##[error]08:43:28.822 WARN: No roslyn issues report not found for this project.
    2018-08-16T07:43:28.8315503Z 08:43:28.822 WARN: No roslyn issues report not found for this project.

I have also a lot of error messages like this :

    2018-08-16T07:43:27.8889117Z ##[error]08:43:27.885 WARN: File 'S:\VSTS\_work\123\s\WSDAL\WSDAL\WSDAL.csproj' is ignored because it doesn't belong to the forced language 'cs'
    2018-08-16T07:43:27.8889699Z 08:43:27.885 WARN: File 'S:\VSTS\_work\123\s\WSDAL\WSDAL\WSDAL.csproj' is ignored because it doesn't belong to the forced language 'cs'
    2018-08-16T07:43:27.8890597Z ##[error]08:43:27.885 WARN: File 'S:\VSTS\_work\123\s\WSDAL\WSDAL.nsi' is ignored because it doesn't belong to the forced language 'cs'
    2018-08-16T07:43:27.8891507Z 
    2018-08-16T07:43:27.8893986Z ##[error]08:43:27.885 WARN: File 'S:\VSTS\_work\123\s\WSDAL\WSDAL.sln' is ignored because it doesn't belong to the forced language 'cs'
    2018-08-16T07:43:27.8896320Z 08:43:27.885 WARN: File 'S:\VSTS\_work\123\s\WSDAL\WSDAL.sln' is ignored because it doesn't belong to the forced language 'cs'

The analyze is launched from VSTS with the SonarQube plugin and with the additional parameters :

    sonar.sources=.
    sonar.verbose=true
    sonar.language=cs
    sonar.dotnet.visualstudio.testProjectPattern=*.Test
    sonar.exclusions=UITests/**,UnitTests/**
    sonar.sourceEncoding=UTF-8

With the "verbose" option, I have these informations :

    2018-08-16T07:43:18.7068176Z ==============================================================================
    2018-08-16T07:43:19.5912144Z [command]C:\Windows\system32\cmd.exe /D /S /C "S:\VSTS\_work\_tasks\SonarQubeAnalyze_6d01813a-9589-4b15-8491-8164aeb38055\4.3.0\sonar-scanner\bin\sonar-scanner.bat"
    2018-08-16T07:43:19.5912599Z INFO: Scanner configuration file: S:\VSTS\_work\_tasks\SonarQubeAnalyze_6d01813a-9589-4b15-8491-8164aeb38055\4.3.0\sonar-scanner\bin\..\conf\sonar-scanner.properties
    2018-08-16T07:43:19.5912936Z INFO: Project root configuration file: NONE
    2018-08-16T07:43:19.5913156Z 08:43:19.166 INFO: SonarQube Scanner 3.2.0.1227
    2018-08-16T07:43:19.5913401Z 08:43:19.166 INFO: Java 1.8.0_181 Oracle Corporation (32-bit)
    2018-08-16T07:43:19.5913632Z 08:43:19.166 INFO: Windows Server 2012 R2 6.3 x86
    2018-08-16T07:43:19.5913862Z 08:43:19.291 DEBUG: keyStore is : 
    2018-08-16T07:43:19.5914074Z 08:43:19.291 DEBUG: keyStore type is : jks
    2018-08-16T07:43:19.5914289Z 08:43:19.291 DEBUG: keyStore provider is : 
    2018-08-16T07:43:19.5915184Z 08:43:19.291 DEBUG: init keystore
    2018-08-16T07:43:19.5915409Z 08:43:19.291 DEBUG: init keymanager of type SunX509
    2018-08-16T07:43:19.5915658Z 08:43:19.432 DEBUG: Create: C:\Users\yannick\.sonar\cache
    2018-08-16T07:43:19.5915896Z 08:43:19.432 INFO: User cache: C:\Users\yannick\.sonar\cache
    2018-08-16T07:43:19.5916159Z 08:43:19.447 DEBUG: Create: C:\Users\yannick\.sonar\cache\_tmp
    2018-08-16T07:43:19.5916398Z 08:43:19.447 DEBUG: Extract sonar-scanner-api-batch in temp...
    2018-08-16T07:43:19.5916636Z 08:43:19.447 DEBUG: Get bootstrap index...
    2018-08-16T07:43:19.5916872Z 08:43:19.447 DEBUG: Download: http://SonarQubeServer:9999/batch/index
    2018-08-16T07:43:19.5917121Z 08:43:19.494 DEBUG: Get bootstrap completed
    2018-08-16T07:43:19.5917341Z 08:43:19.510 DEBUG: Create isolated classloader...
    2018-08-16T07:43:19.5917572Z 08:43:19.510 DEBUG: Start temp cleaning...
    2018-08-16T07:43:19.5917783Z 08:43:19.510 DEBUG: Temp cleaning done
    2018-08-16T07:43:19.5918014Z 08:43:19.510 DEBUG: Execution getVersion
    2018-08-16T07:43:19.5918225Z 08:43:19.525 INFO: SonarQube server 7.2.1
    2018-08-16T07:43:19.5918575Z 08:43:19.525 INFO: Default locale: "en_US", source code encoding: "UTF-8"
    2018-08-16T07:43:19.5918833Z 08:43:19.525 DEBUG: Work directory: S:\VSTS\_work\123\s\.scannerwork
    2018-08-16T07:43:19.5919082Z 08:43:19.525 DEBUG: Execution execute
    2018-08-16T07:43:19.7662081Z 08:43:19.760 INFO: Publish mode
    2018-08-16T07:43:19.9390673Z 08:43:19.932 INFO: Load global settings
    2018-08-16T07:43:20.0043807Z 08:43:19.978 DEBUG: GET 200 http://SonarQubeServer:9999/api/settings/values.protobuf | time=46ms
    2018-08-16T07:43:20.0292709Z 08:43:20.025 INFO: Load global settings (done) | time=93ms
    2018-08-16T07:43:20.0369335Z 08:43:20.025 INFO: Server id: EsA58sX698qSes654w
    2018-08-16T07:43:20.0492805Z 08:43:20.041 INFO: User cache: C:\Users\yannick\.sonar\cache
    2018-08-16T07:43:20.4268501Z 08:43:20.416 INFO: Load/download plugins
    2018-08-16T07:43:20.4271693Z 08:43:20.416 INFO: Load plugins index
    2018-08-16T07:43:20.4353642Z 08:43:20.432 DEBUG: GET 200 http://SonarQubeServer:9999/api/plugins/installed | time=16ms
    2018-08-16T07:43:20.4895763Z 08:43:20.479 INFO: Load plugins index (done) | time=63ms
    2018-08-16T07:43:20.5094751Z 08:43:20.494 INFO: Load/download plugins (done) | time=78ms
    2018-08-16T07:43:20.5569127Z 08:43:20.541 DEBUG: Plugins:
    2018-08-16T07:43:20.5575528Z 08:43:20.541 DEBUG:   * SonarPython 1.10.0.2131 (python)
    2018-08-16T07:43:20.5575997Z 08:43:20.541 DEBUG:   * SonarGo 1.1.0.1612 (go)
    2018-08-16T07:43:20.5576307Z 08:43:20.541 DEBUG:   * Svn 1.8.0.1168 (scmsvn)
    2018-08-16T07:43:20.5576629Z 08:43:20.541 DEBUG:   * SonarJS 4.2.1.6529 (javascript)
    2018-08-16T07:43:20.5576929Z 08:43:20.541 DEBUG:   * SonarC# 7.3.0.5690 (csharp)
    2018-08-16T07:43:20.5577237Z 08:43:20.541 DEBUG:   * SonarJava 5.6.0.15032 (java)
    2018-08-16T07:43:20.5577533Z 08:43:20.541 DEBUG:   * LDAP 2.2.0.608 (ldap)
    2018-08-16T07:43:20.5577842Z 08:43:20.541 DEBUG:   * Git 1.4.1.1128 (scmgit)
    2018-08-16T07:43:20.5578172Z 08:43:20.541 DEBUG:   * SonarFlex 2.4.0.1222 (flex)
    2018-08-16T07:43:20.5578472Z 08:43:20.541 DEBUG:   * SonarXML 1.5.1.1452 (xml)
    2018-08-16T07:43:20.5578780Z 08:43:20.541 DEBUG:   * SonarPHP 2.14.0.3569 (php)
    2018-08-16T07:43:20.5579084Z 08:43:20.541 DEBUG:   * SonarTS 1.7.0.2893 (typescript)
    2018-08-16T07:43:20.5720186Z 08:43:20.557 INFO: Loaded core extensions: 
    2018-08-16T07:43:20.9135176Z 08:43:20.900 INFO: Process project properties
    2018-08-16T07:43:20.9203852Z 08:43:20.916 DEBUG: Process project properties (done) | time=16ms
    2018-08-16T07:43:20.9398896Z 08:43:20.932 INFO: Load project repositories
    2018-08-16T07:43:21.4140553Z 08:43:21.400 DEBUG: GET 200 http://SonarQubeServer:9999/batch/project.protobuf?key=project_key | time=468ms
    2018-08-16T07:43:21.5284145Z 08:43:21.525 INFO: Load project repositories (done) | time=593ms
    2018-08-16T07:43:21.6615142Z 08:43:21.652 DEBUG: Available languages:
    2018-08-16T07:43:21.6616619Z 08:43:21.652 DEBUG:   * Python => "py"
    2018-08-16T07:43:21.6616940Z 08:43:21.652 DEBUG:   * Go => "go"
    2018-08-16T07:43:21.6617249Z 08:43:21.652 DEBUG:   * JavaScript => "js"
    2018-08-16T07:43:21.6617534Z 08:43:21.652 DEBUG:   * C# => "cs"
    2018-08-16T07:43:21.6617827Z 08:43:21.652 DEBUG:   * Java => "java"
    2018-08-16T07:43:21.6618165Z 08:43:21.652 DEBUG:   * Flex => "flex"
    2018-08-16T07:43:21.6618457Z 08:43:21.652 DEBUG:   * XML => "xml"
    2018-08-16T07:43:21.6618733Z 08:43:21.652 DEBUG:   * PHP => "php"
    2018-08-16T07:43:21.6619030Z 08:43:21.652 DEBUG:   * TypeScript => "ts"
    2018-08-16T07:43:21.6684968Z 08:43:21.666 INFO: Load quality profiles
    2018-08-16T07:43:21.6864786Z 08:43:21.682 DEBUG: GET 200 http://SonarQubeServer:9999/api/qualityprofiles/search.protobuf?projectKey=project_key | time=16ms
    2018-08-16T07:43:21.6911948Z 08:43:21.682 INFO: Load quality profiles (done) | time=16ms
    2018-08-16T07:43:21.7055123Z 08:43:21.697 INFO: Load active rules
    2018-08-16T07:43:21.7292078Z 08:43:21.713 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=cs-sonar-way-40297&p=1&ps=500 | time=16ms
    2018-08-16T07:43:21.8274962Z 08:43:21.822 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=java-sonar-way-05385&p=1&ps=500 | time=62ms
    2018-08-16T07:43:21.8578910Z 08:43:21.854 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=TR5ERPD85D3EEE&p=1&ps=500 | time=16ms
    2018-08-16T07:43:21.8703088Z 08:43:21.869 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=ER6DFX8I65d&p=1&ps=500 | time=15ms
    2018-08-16T07:43:21.8942153Z 08:43:21.885 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=DE5DD5OPP67DD&p=1&ps=500 | time=16ms
    2018-08-16T07:43:21.9363457Z 08:43:21.916 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=js-sonar-way-28294&p=1&ps=500 | time=31ms
    2018-08-16T07:43:21.9594956Z 08:43:21.947 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=DDDD5FECDVSDQ&p=1&ps=500 | time=15ms
    2018-08-16T07:43:21.9782101Z 08:43:21.963 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=DSCDSQCDSQCD&p=1&ps=500 | time=16ms
    2018-08-16T07:43:21.9989077Z 08:43:21.994 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,templateKey,params,actives,createdAt&activation=true&qprofile=DSQDFSQCDSQCDS&p=1&ps=500 | time=15ms
    2018-08-16T07:43:22.0113544Z 08:43:22.010 INFO: Load active rules (done) | time=313ms
    2018-08-16T07:43:22.0179051Z 08:43:22.010 INFO: Load metrics repository
    2018-08-16T07:43:22.0256465Z 08:43:22.010 DEBUG: GET 200 http://SonarQubeServer:9999/api/metrics/search?f=name,description,direction,qualitative,custom&ps=500&p=1 | time=0ms
    2018-08-16T07:43:22.0365801Z 08:43:22.025 INFO: Load metrics repository (done) | time=15ms
    2018-08-16T07:43:22.0427992Z 08:43:22.041 DEBUG: SCM Step is disabled by configuration
    2018-08-16T07:43:22.0493558Z 08:43:22.041 INFO: Project key: project_key
    2018-08-16T07:43:22.0494085Z 08:43:22.041 INFO: Project base dir: S:\VSTS\_work\123\s
    2018-08-16T07:43:22.0503378Z 08:43:22.041 DEBUG: Start recursive analysis of project modules
    2018-08-16T07:43:22.0511419Z 08:43:22.041 INFO: -------------  Scan project
    2018-08-16T07:43:22.2015446Z 08:43:22.197 INFO: Load server rules
    2018-08-16T07:43:22.2897220Z 08:43:22.275 DEBUG: GET 200 http://SonarQubeServer:9999/api/rules/list.protobuf | time=78ms
    2018-08-16T07:43:22.3396578Z 08:43:22.307 INFO: Load server rules (done) | time=110ms
    2018-08-16T07:43:22.3479690Z 08:43:22.338 INFO: Base dir: S:\VSTS\_work\123\s
    2018-08-16T07:43:22.3480392Z 08:43:22.338 INFO: Working dir: S:\VSTS\_work\123\s\.scannerwork
    2018-08-16T07:43:22.3491581Z 08:43:22.338 INFO: Source paths: .
    2018-08-16T07:43:22.3493099Z 08:43:22.338 INFO: Source encoding: UTF-8, default locale: en_US
    2018-08-16T07:43:22.4591820Z 08:43:22.447 DEBUG: Declared extensions of language Python were converted to sonar.lang.patterns.py : **/*.py
    2018-08-16T07:43:22.4592818Z 08:43:22.447 DEBUG: Declared extensions of language Go were converted to sonar.lang.patterns.go : **/*.go
    2018-08-16T07:43:22.4593300Z 08:43:22.447 DEBUG: Declared extensions of language JavaScript were converted to sonar.lang.patterns.js : **/*.js,**/*.jsx,**/*.vue
    2018-08-16T07:43:22.4593740Z 08:43:22.447 DEBUG: Declared extensions of language C# were converted to sonar.lang.patterns.cs : **/*.cs
    2018-08-16T07:43:22.4594149Z 08:43:22.447 DEBUG: Declared extensions of language Java were converted to sonar.lang.patterns.java : **/*.java,**/*.jav
    2018-08-16T07:43:22.4594594Z 08:43:22.447 DEBUG: Declared extensions of language Flex were converted to sonar.lang.patterns.flex : **/*.as
    2018-08-16T07:43:22.4594992Z 08:43:22.447 DEBUG: Declared extensions of language XML were converted to sonar.lang.patterns.xml : **/*.xml,**/*.xsd,**/*.xsl
    2018-08-16T07:43:22.4595455Z 08:43:22.447 DEBUG: Declared extensions of language PHP were converted to sonar.lang.patterns.php : **/*.php,**/*.php3,**/*.php4,**/*.php5,**/*.phtml,**/*.inc
    2018-08-16T07:43:22.4595960Z 08:43:22.447 DEBUG: Declared extensions of language TypeScript were converted to sonar.lang.patterns.ts : **/*.ts,**/*.tsx
    2018-08-16T07:43:22.4596303Z 08:43:22.447 INFO: Language is forced to cs
    2018-08-16T07:43:22.4682965Z 08:43:22.463 DEBUG: Initializers : 
    2018-08-16T07:43:22.4691697Z 08:43:22.463 INFO: Index files

I really don't understand what is the problem. I've tried to find a solution on internet, but without success.
Any idea or solution about these issues please ?

**Comment by franciscoarturoalonso** [comment]
Any solution to the last comment/question? I'm having the same issue with the same version and haven't found a solution yet.

Thanks!

**Comment by Evangelink** [comment]
Please create a new issue and post all required information and logs. Even if your issue looks alike we know from past experience that they are rarely exactly the same.
